### PR TITLE
chore: added new fingerprints

### PR DESCRIPTION
**Pull request by akomisarek**
### JIRA link (if applicable) ###

N/A

### Change description ###

Adding new fingerprints to allow security build to pass. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: audit.json [hunk 1]
```diff
@@ -94,6 +96,7 @@
   "10096_Timestamp Disclosure - Unix_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/callback/add-hearing-bookings/about-to-start_POST": "ignore",
   "10096_Timestamp Disclosure - Unix_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/notify-gatekeeper/about-to-start_POST": "ignore",
   "90033_Loosely Scoped Cookie_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/case-submission/mid-event_POST": "ignore",
+  "90033_Loosely Scoped Cookie_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/remove-order/about-to-start_POST": "ignore",
```
Comment (lewisbirks) [inline audit.json:99]: Do you wanna add cookie stuff for the new mid event and about to submit as well?

**Review by tomaszpowroznik — DISMISSED** [review]

File: audit.json (hunk 1 above)
Comment (lewisbirks) [reply audit.json:99]: Just wondering whether we should try and be proactive or reactionary with them

Comment (akomisarek) [reply audit.json:99]: Good point, I was thinking the same as well. I will add.

Comment (akomisarek) [reply audit.json:99]: I think we should be running script just to add everything :P

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**